charts/alfresco-transform-service/templates/_helpers-name.tpl

```diff
@@ -33,6 +33,11 @@
 {{- include "alfresco-transform-service.fullname" $scope }}
 {{- end }}
 
+{{- define "alfresco-transform-service.config-messagebroker.name" -}}
+{{- $scope := (dict "Values" (dict "nameOverride" "messagebroker-configmap" ) "Chart" .Chart "Release" .Release) }}
```

Comment:
I think using "component-objecttype" as names is not very good. It's both reduddant (as you normally know what obj type you're querying) and not self explanatory (here it's not giving any about the fact the cm is used by ats)
I would prefer using something like "message-broker-ats" or "alfresco-transform-service-mq"

Reply:
indeed I can reuse the same named template for deployment/config/secret and have a separated name template only for additional config/secrets